libraries/chain/database.cpp

```diff
@@ -5235,6 +5235,12 @@ void database::apply_hardfork( uint32_t hardfork )
          auto change_request = find< change_recovery_account_request_object, by_account >( STEEM_TREASURY_ACCOUNT );
          if( change_request )
             remove( *change_request );
+
+         modify( get< reward_fund_object, by_name >( STEEM_POST_REWARD_FUND_NAME ), [&]( reward_fund_object& rfo )
+         {
+            rfo.percent_content_rewards = 50 * STEEM_1_PERCENT;
```

Comment:
I was mistaken when we talked. This needs to be 100%.

This is the percent of all content rewards that this reward fund should get. When we implemented reward funds we were planning to have separate reward funds for root posts and comments.